### ISSUE TITLE
Box2D ChainShape is broken

### ISSUE THREAD

**Issue by czyzby** [libgdx/libgdx#4960]
#### Issue details
The latest Box2D version (1.9.7) fails to create correct chain shapes that used to work with previous versions.

#### Reproduction steps/code
```Java
import com.badlogic.gdx.ApplicationAdapter;
import com.badlogic.gdx.math.Vector2;
import com.badlogic.gdx.physics.box2d.BodyDef;
import com.badlogic.gdx.physics.box2d.ChainShape;
import com.badlogic.gdx.physics.box2d.FixtureDef;
import com.badlogic.gdx.physics.box2d.World;

public class Test extends ApplicationAdapter {
	@Override
	public void create () {
		World world = new World(new Vector2(), true);
		BodyDef bodyDef = new BodyDef();
		FixtureDef fixtureDef = new FixtureDef();
		ChainShape shape = new ChainShape();
		float[] vertices = {0f, 0f, 1f, 1f, 0f, 1f};
		shape.createChain(vertices);
		fixtureDef.shape = shape;
		world.createBody(bodyDef).createFixture(fixtureDef);
	}
}
```

This code snippet works correctly on LibGDX 1.9.6, but fails on LibGDX 1.9.7 with a native error.

#### Version of LibGDX and/or relevant dependencies
1.9.7 (LibGDX + Box2D).

#### Stacktrace
```
java: /var/lib/jenkins/workspace/libgdx/extensions/gdx-box2d/gdx-box2d/jni/Box2D/Collision/Shapes/b2ChainShape.cpp:62: void b2ChainShape::CreateChain(const b2Vec2*, int32): Assertion `count >= 2' failed.
```

#### Affected platforms

Note that I checked only on Linux Mint, but other platforms are expected to be affected as well.

- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [ ] Windows
- [x] Linux
- [ ] MacOS

**Comment by Tom-Ski** [comment]
https://github.com/libgdx/libgdx/issues/4940

**Comment by czyzby** [comment]
@Tom-Ski Is a hotfix release planned or should we wait a few months for 1.9.8?

**Comment by Tom-Ski** [comment]
Yup.

**Comment by QStormDS** [comment]
Is there any schedule for the hotfix?

**Comment by Tom-Ski** [comment]
Yes, I wanted to wait for the 2017.3 release of IDEA to solve #4698, but as this has been released without fix we'll have a release this week in the meantime.

**Comment by QStormDS** [comment]
I thought this fix will be included in the libGDX release 1.9.8, as it has been closed 1 month before the release was made. However, it's not included. Will it be included in the next release? Is there any schedule for the next release? Minewhile, I have to stay at 1.9.6. :(

**Comment by Tom-Ski** [comment]
This issue is a duplicate. See issue referred too for the tracking of this issue.
